src/App/PropertyExpressionEngine.cpp

```diff
@@ -236,6 +256,10 @@ void PropertyExpressionEngine::updateHiddenReference(const std::string &key) {
     }
 }
 
+void PropertyExpressionEngine::varSetChanged(const App::DocumentObject &, const App::Property &) {
+    dynamic_cast<DocumentObject*>(getContainer())->touch(false);
```

Comment:
This is prone to segfault if the container isn't a DocumentObject.

Reply:
You're right. Resolved with the recent commits.